build_tools/github/test_wheels.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+set -x
+
+if [[ "$OSTYPE" != "linux-gnu" ]]; then
+    # The Linux test environment is run in a Docker container and
+    # it is not possible to copy the test configuration file (yet)
```

Comment:
WDYT about this situation (that is, we cannot copy the test configuration file in Linux)?

Reply:
I think moving the contents of `conftest.py` into the `sklearn` package makes sense.

Reply:
I will open an issue for that ASAP.